tests/client/test_save_copy_delete_judgment.py

```diff
@@ -143,9 +145,10 @@ def test_insert_document_xml(self):
                 ),
             }
             self.client.insert_document_xml(
-                uri,
-                document_xml,
-                VersionAnnotation(
+                document_uri=uri,
+                document_xml=document_xml,
+                document_type=Judgment,
```

Comment:
This feels a little like it might want to be a Judgment.insert_document_xml in future.